.github/workflows/stylelint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint CSS
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install stylelint
+        run: npm install -g stylelint stylelint-config-standard
+
+      - name: Run stylelint
+        run: stylelint "**/*.css"
```

Comment:
Remove the double quotes. They will treat the expression `**/*.css` as literal. Unless the `stylelint` command interprets a glob expression?

Reply:
You're right, I removed the double quotes!